.github/workflows/ci.yml

```diff
@@ -14,6 +14,7 @@ jobs:
     strategy:
       matrix:
         node: [16]
+        npm: [8.5.x]
```

Comment:
I wouldn't expect it's necessary to pin npm here or install it below... Node 16 should already come with a version of npm 8.x.

Reply:
CI is installing version 8.11.x and many tests are failing on it. That's why after collaboration with arch-bom we pinned it in our PRs.

Reply:
Not sure I follow the part about tests failing.  All the checks on this repo appear to be passing with this PR as-is without pinning the npm version, right?  Going to approve at any rate. 👍🏻